### PR TITLE
Add iam

### PR DESCRIPTION
**Pull request by krisiye**
- Adds IAM to the default concourse helm 3 resource.
- Updates to Readme

### CODE REVIEW

**Review by dabigc — APPROVED** [review]
LGTM

File: assets/common.sh [hunk 1]
```diff
@@ -13,9 +13,26 @@ setup_kubernetes() {
...
+  # check creds
+  aws configure list
+  aws sts get-caller-identity
+
   absolute_kubeconfig_path="${source}/${kubeconfig_path}"
   if [ -f "$absolute_kubeconfig_path" ]; then
     cp "$absolute_kubeconfig_path" "/root/.kube/config"
+    export KUBECONFIG=/root/.kube/config
+
+    echo "AWS_ACCESS_KEY_ID: $AWS_ACCESS_KEY_ID"
```
Comment (etotten) [inline assets/common.sh:32]: These were for testing I assume...should they go away?

Comment (krisiye) [reply assets/common.sh:32]: The default is that any echo's will be disabled. These could remain if you like.. will only show up if trace is enabled on the resource. Thoughts?

**Review by etotten — APPROVED** [review]
LGTM

**Pull request merged**